src/toolboxes/background/UI.tsx

```diff
@@ -46,50 +46,120 @@ const BackgroundUI = (props: Props): ReactElement => {
   const controls = props.channels.map((channel, i) => (
     <Checkbox
       checked={channel}
+      icon={
+        <SVG
+          src="./src/assets/not-selected-tickbox-icon.svg"
+          width="18px"
+          height="auto"
+        />
+      }
+      checkedIcon={
+        <SVG
+          src="./src/assets/selected-tickbox-icon.svg"
+          width="18px"
+          height="auto"
+        />
+      }
+      style={{ marginLeft: "104px" }}
       onChange={() => {
         props.toggleChannelAtIndex(i);
       }}
     />
   ));
 
   return (
-    <Accordion expanded={props.expanded} onChange={props.onChange}>
-      <AccordionSummary expandIcon={<ExpandMore />} id="background-toolbox">
-        <Typography>Image</Typography>
-      </AccordionSummary>
-      <AccordionDetails>
-        <Grid container spacing={0} justify="center" wrap="nowrap">
-          <Grid item style={{ width: "85%", position: "relative" }}>
+    <>
+      <Popover
+        open={props.open}
+        anchorEl={props.anchorEl}
+        onClose={props.onClose}
+      >
+        {props.buttonClicked === "Contrast" && (
+          <div
+            style={{
+              width: "63px",
+              height: "335px",
+            }}
+          >
             <BaseSlider
               value={background.contrast}
               config={SLIDER_CONFIG[Sliders.contrast]}
               onChange={() => changeContrast}
+              slider="contrast"
             />
+          </div>
+        )}
+
+        {props.buttonClicked === "Brightness" && (
+          <div
+            style={{
```

Comment:
Again, no in-line stylings please - I would guess this would be the same for all sliders in annotate? So a class seems reasonable.